src/resources/app-fr.json

```diff
@@ -414,8 +415,9 @@
   "exitingAppletHint": "Tout le contenu du panier sera ajouté au contenu de l'Applet existante",
   "export": "Exporter",
   "exportData": "Exporter les données",
+  "exportDataForApplet": "Exporter les données pour {{name}}",
   "exportDefaultSchedule": "Exporter le programme par défaut",
-  "exportDescription": "Exporter les données pour la plage de dates sélectionnée",
+  "exportDescription": "Choisissez ce que vous souhaitez inclure :",
```

Comment:
```suggestion
  "exportDescription": "Choisissez ce que vous souhaitez inclure:",
```

Reply:
In French a space is added before `:` (though not in Canadian French) so I will leave this as is